__tests__/service/anonymousMessageRepo.test.ts

```diff
@@ -60,7 +60,7 @@ describe('TESTING anonymousMessageRepo', () => {
 
       // Assertion
       expect(result).toEqual([]);
-      expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat/anonymous');
+      expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat-v2/anonymous');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change from '/feeds/feed-chat/anonymous' to '/feeds/feed-chat-v2/anonymous' is in line with the PR's objective. Again, make sure the new endpoint '/feeds/feed-chat-v2/anonymous' is properly set up and tested on the server side.
```diff
- expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat/anonymous');
+ expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat-v2/anonymous');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->